src/main.rs

```diff
@@ -159,6 +166,15 @@ fn dump_setting(path: &PathBuf) -> rekordcrate::Result<()> {
     Ok(())
 }
 
+fn dump_xml(path: &PathBuf) -> rekordcrate::Result<()> {
+    let file = std::fs::File::open(path)?;
+    let reader = std::io::BufReader::new(file);
+    let document: Document = quick_xml::de::from_reader(reader).expect("failed to deserialize XML");
```

Comment:
does this need to panic? Would've something like this make more sense?
```suggestion
    let document: Document = quick_xml::de::from_reader(reader).map_err(|_| "failed to deserialize XML")?;
```

Reply:
Yeah makes sense.